_includes/js.html

```diff
@@ -7,6 +7,7 @@
     <!-- Plugin JavaScript -->
     <script src="{{ "/js/jquery.easing.min.js" | prepend: site.baseurl }}"></script>
     <script src="{{ "/js/classie.js" | prepend: site.baseurl }}"></script>
+    
```

Comment:
@jaimemendozadev also so I know going forward, would you like me to modify your commits or comment here and leave it with you make changes?